config/config-test.ts

```diff
@@ -1,33 +1,57 @@
-
 const network = '4002';
 const isDev = process.env.NODE_ENV === 'development';
 const isLive = 'true';
 const isTestnet = true;
 
+const WFTMAddress = "0xd0011de099e514c2094a510dd0109f91bf8791fa";
+const GTONAddress = "0xc4d0a76BA5909c8e764B67aCF7360f843FbACB2d";
+
+const spiritswaprouteraddress = '0x3c4925b50e337aecc2cf4b9e4767b43dcfbad286';
+const spiritswappooladdress = '0x131ee332febf8d2aa5f18f0d2c444e99411462e5';
```

Comment:
name constants in capitalized snake case.

Reply:
Will fix it in the upcoming refactoring PR, @GooGrand please take a note